awesomeConfig.cmake

```diff
@@ -37,8 +37,7 @@ endmacro()
 
 a_find_program(GIT_EXECUTABLE git FALSE)
 # programs needed for man pages
-a_find_program(ASCIIDOC_EXECUTABLE asciidoc FALSE)
-a_find_program(XMLTO_EXECUTABLE xmlto FALSE)
+a_find_program(ASCIIDOC_EXECUTABLE asciidoctor FALSE)
```

Comment:
Shouldn't this be renamed to `ASCIIDOCTOR_EXECUTABLE`?
Although keeping it picks up any custom user config/setting.

Reply:
i think that's fine to rename since any custom build which was relying on ASCIIDOC_EXUTABLE could also break because of the removing of XMLTO_EXECUTABLE

Reply:
Thanks, I updated the variable name